android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableView.java

```diff
@@ -265,6 +281,16 @@ public void update()
 			}
 		}
 
+		// Add placeholder item for TableView footer.
+		if (hasFooter) {
+			final TableViewRowProxy row = new TableViewRowProxy();
+
+			row.getProperties().put(TiC.PROPERTY_FOOTER, properties.get(TiC.PROPERTY_FOOTER_TITLE));
```

Comment:
Just to be sure: It has to be `PROPERTY_FOOTER ` instead of `PROPERTY_FOOTER_TITLE`? ✌️

Reply:
Yes, this is correct. `TableViewRow` requires `PROPERTY_FOOTER` for the title, where `TableView` requires `PROPERTY_FOOTER_TITLE`.

Maybe I should use this opportunity to deprecate `PROPERTY_FOOTER` in favor of `PROPERTY_FOOTER_TITLE` for parity